app/services/pii/address_data.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/MutableConstant
+module Pii
+  AddressData = RedactedData.define(:state, :zipcode, :city, :address1, :address2)
```

Comment:
Could we just call this `Address`? I think `Pii::Address` flows a little nicer.

Reply:
Yeah, I went back and forth but let me just change it to `Address` since that does look better.